media/public/include/IMediaPipeline.h

```diff
@@ -923,6 +923,16 @@ class IMediaPipeline
      */
     virtual bool removeSource(int32_t id) = 0;
 
+    /**
+     * @brief Notifies Rialto Server that all sources were attached
+     *
+     * This method is called by Rialto Client to notify the RialtoServer
+     * that all sources were attached.
```

Comment:
We should make it explicit that this must be called otherwise streaming cannot commence